ocs_ci/deployment/ibmcloud.py

```diff
@@ -157,11 +158,22 @@ def deploy_ocp(self, log_cli_level="DEBUG"):
             raise UnsupportedPlatformVersionError(
                 "IBM Cloud IPI deployments are only supported on OCP versions >= 4.10"
             )
+        # By default, IBM cloud has load balancer limit of 50 per region.
+        # switch to us-south, if current load balancers are more than 45.
+        # https://cloud.ibm.com/docs/vpc?topic=vpc-quotas
+        ibmcloud.login()
+        if config.ENV_DATA.get("enable_region_dynamic_switching"):
+            lb_count = self.get_load_balancers_count()
+            if lb_count > (IBM_CLOUD_LOAD_BALANCER_QUOTA - 5):
+                logger.info(
+                    "Switching region to us-south due to lack of load balancers"
```

Comment:
We can do it in a follow up PR - instead of always starting with us-east, how about randomly picking out of south or east?

Reply:
yes, we can do that as well.